demo/admin/src/dam/getRandomUnsplashImage.ts

```diff
@@ -23,7 +35,7 @@ function extractFileNameFromUrl(url: string): string {
 }
 
 export async function getRandomUnsplashImage(): Promise<UnsplashImage> {
-    const imageUrl = "https://source.unsplash.com/all/";
+    const imageUrl = "https://picsum.photos/1920/1080"; // https://picsum.photos/ uses unsplashed images
```

Comment:
off-topic, but found this which uses unsplashed images too but works fine, seems like a good replacement

Reply:
In the demo I added the data to the file like this:
![Screenshot 2024-12-06 at 14 07 53](https://github.com/user-attachments/assets/dd0973bb-cc3d-4100-82eb-3b69c49355f2)

Reply:
Nice find, we should switch that